### DISCUSSION TITLE
config/notes/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#140] [docs-comment]
# config/notes/

概述 主题默认将 sourceDir 目录下的 notes 目录作为 文档/知识笔记 的存放目录， 该目录下的所有文件都会被排除在 博客文章之外。 默认配置如下： 如果启用了 多语言配置， 你 也可以在 locales 字段中 分别配置 notes 配置 自动生成侧边栏 当配置某个 笔记 的 sidebar 为 auto 时，会自动根据目录结构生成侧边...

https://plume.pengzhanbo.cn/config/notes/

**Comment by tianhairui** [comment]
我按照指南和配置，已完成了`主题`、`博客`和`notes`的相关简单配置，并通过点击`navbar`定义的`note`也可以跳转至对应的`md`文档，一切都正常。目前遇到的困难是，跳转到`md`文档后，没有生成侧边栏`sidebar`，尝试了好几个地方添加，并不起作用，是不是我遗漏了哪里？貌似没找到`sidebar`的配置示例。

**Reply by pengzhanbo** [reply]
博客类的文章是不会生成 sidebar 的。 当前有两种方式用于配置 sidebar。
- [notes 配置](https://theme-plume.vuejs.press/config/notes/#%E9%85%8D%E7%BD%AE) 中，根据每个单独的 note 有一个 `sidebar` 配置项。 
- 主题配置项有一个 `sidebar` 的选项，但我目前还没有把它更新到文档说明中。

**Reply by pengzhanbo** [reply]
关于 侧边栏 配置，我最近抽空 写到一个 单独的 md 文件中进行详细描述吧。

**Reply by tianhairui** [reply]
十分感谢

**Comment by bawanchun** [comment]
博主你好，感谢你的作品，我也出现了上个评论的情况，关于notes的侧边栏的使用说明能否像https://theme-plume.vuejs.press/guide/custom-home/一样，有一些例子说明，再次感谢

**Reply by pengzhanbo** [reply]
我找个时间写个详细的说明吧

**Reply by pengzhanbo** [reply]
@blue997 https://theme-plume.vuejs.press/guide/document/ 详细使用文档已更新。

**Reply by bawanchun** [reply]
xiexie

**Comment by Niceies** [comment]
设置sidebar: 'auto',情况下怎么设置collapsed?，希望所有页面都能默认折叠
